CMakeLists.txt

```diff
@@ -1,76 +1,80 @@
 cmake_minimum_required(VERSION 2.8.12.2 FATAL_ERROR)
-
 project(prometheus-cpp)
+
 enable_testing()
-include(GNUInstallDirs)
 
+include(GNUInstallDirs)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_LIST_DIR}/cmake)
 
 if(NOT DEFINED CMAKE_CXX_STANDARD AND UNIX)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+	set(CMAKE_CXX_STANDARD 11)
 endif()
 
+OPTION(BUILD_CIVETWEB "Embed Civet webserver in prometheus" ON)
```

Comment:
please use lowercase command names. Lowercase is also what CMake itself uses nowadays.

Reply:
Fixed.